WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -476,7 +483,12 @@ - (void)showTabForIndex:(NSInteger)tabIndex
 - (NSInteger)adjustedTabIndex:(NSInteger)tabIndex toTabType:(BOOL)toTabType {
     //TODO: Remove this change once `floatingCreateButton` feature flag is enabled
     if ([Feature enabled:FeatureFlagFloatingCreateButton] && tabIndex > WPTabReader) {
-        return tabIndex + (toTabType ? -1 : 1); // Adjust the index if we are hiding the new post button
+        if ([Feature enabled:FeatureFlagMeMove]) {
```

Comment:
I wonder if this should be separate. It might not be out of the question for one of these features to ship without the other.

```swift
NSInteger tabOffset = 0;
if ([Feature enabled:FeatureFlagFloatingCreateButton] && tabIndex > WPTabReader) {
    tabOffset += 1;
}
if ([Feature enabled:FeatureFlagMeMove] && tabIndex > WPTabNewPost) {
    tabOffset += 1;
}
return tabIndex + (toTabType ? -tabOffset : tabOffset);
```

_I didn't actually test this, it's just an example._

Reply:
I think they will likely be shipped together but yeah, this reads a lot better. I just tested it and seems to work